modules/monitoring-assigning-tolerations-to-monitoring-components.adoc

```diff
@@ -0,0 +1,125 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/configuring-the-monitoring-stack.adoc
+
+[id="assigning-tolerations-to-monitoring-components_{context}"]
+= Assigning tolerations to monitoring components
+
+[role="_abstract"]
+You can assign tolerations to any of the monitoring stack components to enable moving them to tainted nodes.
+
+.Prerequisites
+
+* *If you are configuring core {product-title} monitoring components*:
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
```

Comment:
No applicable for OSD/ROSA

Reply:
Done.

Reply:
I assume this section of the first step in the procedure also should be removed. I have done so. Please verify.